firmware/projects/TMS/main.cc

```diff
@@ -1,12 +1,18 @@
 /// @author Blake Freer
 /// @date 2023-11-18
 
+#include <sys/_stdint.h>
```

Comment:
remove this, clangd likes to add it but cstdint provides the necessary defs

Reply:
good catch, yea i wonder if theres a way to catch these